website/content/en/blog/the-economics-of-the-cncf-platform-engineering-model.md

```diff
@@ -0,0 +1,84 @@
+---
+date: '2023-12-07T21:18:09+01:00'
+draft: true
+title: 'Platform Engineering Through the Lens of Two-Sided Markets'
+author: 'Nagy Viktor'
+---
+
+_Disclaimer: The author is employed by GitLab. The article has benefited from constructive feedback from the CNCF community, with special thanks to Abby Bangser, Lou Bichard, and Colin Griffin for their valuable insights._
+
+## TL;DR
+
+Unveiling the parallels between the challenges of Internal Developer Platforms and the dynamics of two-sided markets, this article explores the strategic decision-making process when applying the CNCF Platform Engineering Maturity Model. The key takeaway emphasizes the criticality of solving the right problems first, triggering a self-sustaining value loop as the platform matures. Additionally, the article sheds light on the challenges associated with switching between platforms and underscores the importance of aligning functionality expectations with market dynamics.
```

Comment:
Having never heard of two-sided markets before, I am suprised you don't explain what it is. I to Google, then squint to see how its comparable and it feels a bit contrived